documentation/blog/2023-09-20-runes.md

```diff
@@ -74,9 +76,10 @@ Well, no. The reality is that as applications grow in complexity, figuring out w
 
 ## Beyond components
 
-With runes, reactivity extends beyond the boundaries of your `.svelte` files. Suppose we wanted to encapsulate our counter logic in a way that could be reused between components. Today, you would use a [custom store](https://learn.svelte.dev/tutorial/custom-stores) in a `.js` or `.ts` file:
+With runes, reactivity extends beyond the boundaries of your `.svelte` files. Suppose we wanted to encapsulate our counter logic in a way that could be reused between components. Today, you would use a [custom store](https://learn.svelte.dev/tutorial/custom-stores) in a `.svelte.js` or `.svelte.ts` file:
```

Comment:
A custom store wouldn't be in a `.svelte.js` file - this sentence is talking about how you'd do this in Svelte 4.

Reply:
Oh yeah good catch 😅